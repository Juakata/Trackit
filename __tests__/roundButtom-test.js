import React from 'react';
import renderer from 'react-test-renderer';

import RoundButton from '../src/components/roundButton';

it('RoundButton snapShot', () => {
  const snap = renderer.create(
    <RoundButton title="Test" color="#FFF" backgroundColor="#000" onClick={jest.fn()} />,
  ).toJSON();
  expect(snap).toMatchSnapshot();
});
