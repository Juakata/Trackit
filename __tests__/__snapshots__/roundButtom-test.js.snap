// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RoundButton snapShot 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "borderRadius": 62,
        "borderWidth": 2,
        "height": 105,
        "justifyContent": "center",
        "width": 105,
      },
      Object {
        "borderColor": "#FFF",
      },
    ]
  }
>
  <View
    accessible={true}
    focusable={true}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#000",
        "borderRadius": 60,
        "height": 100,
        "justifyContent": "center",
        "opacity": 1,
        "width": 100,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#ffffff",
            "fontSize": 16,
            "textAlign": "center",
          },
          Object {
            "color": "#FFF",
          },
        ]
      }
    >
      Test
    </Text>
  </View>
</View>
`;
